kobo/apps/project_ownership/views/transfer.py

```diff
@@ -1,76 +1,60 @@
 from django.db.models import Prefetch
 from django.shortcuts import Http404
-from drf_spectacular.utils import extend_schema
+from drf_spectacular.utils import extend_schema, extend_schema_view
 from rest_framework import viewsets
+from rest_framework.renderers import JSONRenderer
 
 from kpi.permissions import IsAuthenticated
+from kpi.utils.schema_extensions.markdown import read_md
+from kpi.utils.schema_extensions.response import open_api_200_ok_response
 from ..models import Transfer, TransferStatus
+from ..schema_extensions.v2.project_ownership.transfers.serializers import \
+    TransferListResponse
 from ..serializers import TransferDetailSerializer
 
 
 @extend_schema(
-    tags=['transfers'],
+    tags=['Transfers'],
+)
+@extend_schema_view(
+    list=extend_schema(
+        description=read_md('project_ownership', 'transfers/list.md'),
+        responses=open_api_200_ok_response(
+            TransferListResponse,
+            validate_payload=False,
+            require_auth=False,
+        )
+    ),
+    retrieve=extend_schema(
+        description=read_md('project_ownership', 'transfers/retrieve.md'),
+        responses=open_api_200_ok_response(
+            TransferListResponse,
+            validate_payload=False,
+            require_auth=False,
```

Comment:
Missing documentation for 401 errors (when trying to access the endpoint but not logged in):

<img width="1165" height="661" alt="Image" src="https://github.com/user-attachments/assets/d3ffe595-43a3-4588-9493-410e6d08e29e" />